src/components/molecules/UserReactions/UserReactions.tsx

```diff
@@ -43,47 +36,41 @@ export const UserReactions: React.FC<UserReactionsProps> = ({
     userId,
   });
 
-  const { renderedEmojiReactions, userShoutout } = useMemo(() => {
-    const userUniqueEmojiReactions = userReactions.reduce(
-      uniqueEmojiReactionsDataMapReducer,
-      new Map()
+  const { userUniqueEmojiReactions, userShoutout } = useMemo(() => {
+    const userUniqueEmojiReactions = Array.from(
+      userReactions
+        .reduce(uniqueEmojiReactionsDataMapReducer, new Map())
+        .values()
     );
 
-    const renderedEmojiReactions = Array.from(
```

Comment:
I wanted to keep the `useMemo` clean and focused on the calculation (I doubt it's worth having it here, but still). The JSX is moved onto the main section, making this component more readable